sympy/printing/latex.py

```diff
@@ -1611,6 +1611,12 @@ def _print_Identity(self, I):
         return r"\mathbb{I}" if self._settings[
             'mat_symbol_style'] == 'plain' else r"\mathbf{I}"
 
+    def _print_SingleEntryMatrix(self, expr):
+        _, _, i, j = expr.args
+        if self._settings['mat_symbol_style'] == 'plain':
+            return r"\mathbb{J}" + "^{%s, %s}" % (i, j)
+        return r"\mathbf{J}" + "^{%s, %s}" % (i, j)
```

Comment:
I've usually seen the letter `e` used for this. See for instance https://en.wikipedia.org/wiki/Standard_basis

Reply:
Alternately `[\delta_{i,j}]_{n,m}` which is more verbose but includes the shape information. See https://en.wikipedia.org/wiki/Matrix_(mathematics)#Notation (I haven't personally seen this notation used very often but it seems somewhat self-evident what it means).